core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -1264,6 +1264,14 @@
 
   public static final String ENABLE_HIVE_SCHEMA_META_STORE_DEFAULT = "false";
 
+  /**
+   * There is more often that in production uses different drivers for load and queries. So in case
+   * of load driver user should set this property to enable loader specific clean up.
+   */
+  public static final String TABLE_LOADER_DRIVER = "spark.carbon.table.loader.driver";
```

Comment:
I think this property not just for loading, any transactional operation should use this driver. So can you rename it?